src/components/AllServices/allServicesLinks.ts

```diff
@@ -0,0 +1,377 @@
+import AllServicesIcons from './AllServicesIcons';
+
+export type AllServicesLink = {
+  href: string;
+  title: string;
+  isExternal?: boolean;
+};
+export type AllServicesGroup = {
+  isGroup: true;
+  title: string;
+  links: AllServicesLink[];
+};
+export type AllServicesSection = {
+  icon: keyof typeof AllServicesIcons;
+  title: string;
+  description: string;
+  links: (AllServicesLink | AllServicesGroup)[];
+};
+
+const allServicesLinks: AllServicesSection[] = [
```

Comment:
Any thoughts on loading this from an external source? It might be easier to version control separately rather than binding it to a chrome release.

Reply:
Yeah, thats a good point. Currently, we reference it with the navigation files, but I agree the structure should be extracted into another file, similar to what we do with the navigation files.

Reply:
Sure we don't have to address this right now, just a thought for the future.